SimpleROSArm/Assets/Prefabs/ur5e.prefab

```diff
@@ -899,9 +900,23 @@ MonoBehaviour:
   - 63
   - -15
   - 10
-  CurrentAnglesDeg: []
-  CurrJointVelDeg: []
+  CurrentAnglesRad: []
+  CurrJointVelRad: []
   GoalReached: 0
+--- !u!114 &8330969473866756538
+MonoBehaviour:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 2535298805222998953}
+  m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_Script: {fileID: 11500000, guid: feb3bb6f28a2ea5d5b66afdb338153c5, type: 3}
+  m_Name: 
+  m_EditorClassIdentifier: 
+  TopicName: sensor_msgs/JointState
```

Comment:
Update topic name to match value in the scene (default topic the robot_state_publisher is listening for).